### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by abnegate**
## What does this PR do?
- Fixes between query output

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: CHANGELOG.md
Comment (christyjacob4) [inline CHANGELOG.md:1]: we need to add the changelog for dart and flutter manually

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
changelog needs to be updated

File: lib/src/client_browser.dart [hunk 1]
```diff
@@ -156,7 +156,7 @@ class ClientBrowser extends ClientBase with ClientMixin {
 
     while (offset < size) {
       var chunk;
-      final end = min(offset + CHUNK_SIZE - 1, size - 1);
+      final end = min(offset + CHUNK_SIZE, size);
```
Comment (stnguyen90) [inline lib/src/client_browser.dart:159]: @lohanidamodar do we really want to make this change?

File: lib/src/client_io.dart [hunk 2]
```diff
@@ -283,7 +283,7 @@ class ClientIO extends ClientBase with ClientMixin {
     while (offset < size) {
       List<int> chunk = [];
       if (file.bytes != null) {
-        final end = min(offset + CHUNK_SIZE - 1, size - 1);
+        final end = min(offset + CHUNK_SIZE, size);
```
Comment (stnguyen90) [inline lib/src/client_io.dart:286]: @lohanidamodar do we really want to make this change?

File: lib/src/client_browser.dart (hunk 1 above)
Comment (lohanidamodar) [reply lib/src/client_browser.dart:159]: Yes

File: lib/src/client_io.dart (hunk 2 above)
Comment (lohanidamodar) [reply lib/src/client_io.dart:286]: yes

**Review by lohanidamodar — APPROVED** [review]

**Pull request merged**